superset/db_engine_specs/duckdb.py

```diff
@@ -81,9 +84,28 @@ def get_table_names(
     ) -> set[str]:
         return set(inspector.get_table_names(schema))
 
+    @staticmethod
+    def get_extra_params(database: Database) -> dict[str, Any]:
+        """
+        Add a user agent to be used in the requests.
+        """
+        extra: dict[str, Any] = BaseEngineSpec.get_extra_params(database)
```

Comment:
```suggestion
        extra: dict[str, Any] = super().get_extra_params(database)
```

Reply:
This doesn't work unfortunately because it's a `staticmethod` and there is no reference to `self`.